content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/give-feedback-effectively-without-creating-an-unmanageable-workload.md

```diff
@@ -0,0 +1,220 @@
+      </p>
+      <p>
+        Thanks to their approach, teachers are innovative in how they provide feedback. They’re empowered to:
+        <ul>
+          <li>give feedback and mark in a way that best suits their subject and pupils</li>
+          <li>use a wide variety of feedback strategies such as verbal feedback and live marking</li>
+      </p>
+      <p>
+        Teachers are also more selective in what, why and how they mark. They:
+        <ul>
+          <li>focus on what makes a difference in written marking</li>
+          <li>mark for pupils - not leaders</li>
+        </ul>
+      </p>
+      <p>
+      Workload has also reduced and there is a better balance between time spent on planning and marking. Pupils are more actively engaged in checking their own work and responding to feedback.
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Executive Principal Lynn James
+
+{inset-text}
+We have reduced the pressures which increase teacher anxiety, ensuring teachers can excel and focus on what makes the biggest difference. 
+
+Our quality assurance process removes graded lesson observations and instead evaluates how well a subject’s curriculum secures pupils’ learning.  
+
+Pupils’ books become a reflection of effective learning, not a check of marking compliance. 
+{/inset-text}
+
+## Our approach to feedback and marking 
+
+We believe that teachers should spend more time planning how to deliver their curriculum than marking as this has a greater impact on pupils’ learning.  
+
+We: 
+
+* remove unnecessary marking 
+
+* use a range of marking methodologies - one style of marking does not suit all purposes and all pupils 
+
+There are many effective forms of feedback, and we use a variety of methods. We believe that: 
+
+* feedback should be a regular, but not onerous, practice 
+
+* teachers should select the best feedback method available to impact positively on pupil learning and future performance 
+
+We set clear and manageable expectations about how teachers can approach: 
+
+* feedback in lessons 
+
+* marking (for example, how regular it is) 
+
+* identifying mistakes and errors 
+
+* peer and self assessment 
+
+Department leads quality assure marking and feedback in discussions with teams. 
+
+## Use immediate feedback strategies in lessons  
+
+We give feedback as close as possible to the point of action, preferably during the lesson. This is more effective than comments given later. 
+
+We use different types of immediate feedback strategies in lessons. Their usage is evident: 
+
+* in pupils’ books due to improved work 
+
+* by pupils’ conversations about their lessons 
+
+* by observing teaching over time 
+
+### Verbal feedback 
+
+Verbal feedback should be common practice in lessons, and you do not need to record when you’re using it - unless it supports the pupil in their learning. 
+
+ 
+
```

Comment:
Extra lines

Reply:
There are quite a few extra lines below as well that could be tidied up. Not strictly an issue, just makes it cleaner to read!